code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -267,20 +267,17 @@
 	var/mob/living/M = A.affected_mob
 	if(HAS_TRAIT(M, TRAIT_DEATHCOMA))
 		return power
-	else if(M.IsUnconscious() || M.stat == UNCONSCIOUS)
+	else if(M.stat == UNCONSCIOUS)
 		return power * 0.9
 	else if(M.stat == SOFT_CRIT)
 		return power * 0.5
-	else if(M.IsSleeping())
-		return power * 0.25

```

Comment:
This should be replaced by a sleeping status effect check and put above the unconscious check

Reply:
This check was never true because of the unconscious check before. The idea here was not to change behavior.
Why would the disease heal less if you are voluntarily sleeping, though?

Reply:
If it was never true it was clearly a bug.

I made it that way to make it harder to use the symptom as a "free" heal, since it's meant to only work in dire conditions. Initially sleeping gave nothing, but i added it as an option since it is a kind of unconsciousness.
Lore-wise, it'd be explained by the fact that the virus needs the body in a particular coma-like state, and sleep is not deep enough for optimal healing. Unconsciousness allows the virus to induce a near-coma, and critical conditions makes it kick in at full strength.

Reply:
Change done, bug fixed.